pkg/dns/target_test.go

```diff
@@ -1,3 +1,5 @@
+//go:build unit
+
 package dns
 
 import (
```

Comment:
For this file - hardcoded names of the CRs - and the clusters array seems to consist of the same elements shuffled around. There must be a way to define each cluster once and then throw it into the test case